reconcile/utils/runtime/runner.py

```diff
@@ -0,0 +1,200 @@
+import logging
+import sys
+from dataclasses import dataclass
+from typing import (
+    Any,
+    Optional,
+)
+
+from sretoolbox.utils import threaded as sretoolbox_threaded
+
+from reconcile.utils import gql
+from reconcile.utils.runtime.desired_state_diff import (
+    DesiredStateDiff,
+    build_desired_state_diff,
+)
+from reconcile.utils.runtime.integration import QontractReconcileIntegration
+
+
+@dataclass
+class IntegrationRunConfiguration:
```

Comment:
Docstring cover purpose and what each member is used for?

Our CI isn't enforcing this, so I'm not strongly opinionated, but a standard way we can doc these parameters to make them distinct from general comments: https://peps.python.org/pep-0224/

Reply:
I was not aware of attribute docstrings. TIL. I will add respective descriptions.